ons_alpha/utils/jinja2tags.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, environment):
             {
                 "social_text": social_text,
                 "social_image": social_image,
+                "setAttributes": set_attributes_filter,
```

Comment:
This looks unused. Is it a remnant of an earlier implementation?

Reply:
No, this was necessary as otherwise there's an exception (perhaps a NameError?) for the undefined function.